grammars/build.gradle

```diff
@@ -15,5 +15,5 @@ classes {
 
 dependencies {
   antlr "org.antlr:antlr4:${antlrVersion}"
-  api 'org.antlr:antlr4-runtime:4.9.3'
+  api 'org.antlr:antlr4-runtime:4.11.0'
```

Comment:
See previous comment: 4.11.1 has also been released, can we update to that version directly?

Reply:
done :)